frontend/src/lib/utils/candidateStore.ts

```diff
@@ -76,20 +88,36 @@ const logOut = async () => {
   localStorage.clear();
 };
 
-const loadAnswerData = async () => {
-  const answers = await getExistingAnswers();
+const loadOpinionAnswerData = async () => {
+  const answers = await getExistingOpinionAnswers();
 
   if (!answers) return;
```

Comment:
I think this and the other load functions below should throw if no data is found, unless there's a case when that should happen…